evalml/tests/problem_type_tests/test_problem_types.py

```diff
@@ -95,6 +95,14 @@ def test_detect_problem_type_regression():
     assert detect_problem_type(y_float) == ProblemTypes.REGRESSION
 
 
+def test_numeric_extensions():
+    y_Int64 = pd.Series([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12], dtype='Int64')
+    y_Int64_null = pd.Series([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, None], dtype='Int64')
```

Comment:
Does this change if you specify `np.nan` instead of `None`? If numpy/pandas parse these to the same thing, no need to add test coverage; if they're different, we should add coverage.

Reply:
Since I use `pd.dropna()` in the beginning, all NaN/None/null values are treated the same.